changedetectionio/tests/test_notification.py

```diff
@@ -68,6 +70,14 @@ def test_check_notification(client, live_server):
     # Give the thread time to pick up the first version
     time.sleep(3)
 
+    testimage = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII='
+    # Write the last screenshot png
+    datastore = '/tmp/changedetection.io/changedetectionio/test-datastore/'
+
+    for datastoreelement in glob(datastore + "*/"):
```

Comment:
why glob? why not just write to the file directly at `test-datastore/the-image.jpg` ??

Reply:
Because you generate a folder with the uuid of the `Watch`

Reply:
use this to get the UUID

```
from ..util import  extract_UUID_from_client
uuid = extract_UUID_from_client(client)